src/status_im/ui/screens/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,55 @@
+(ns status-im.ui.screens.activity-center.notification.mentions.view
+  (:require [quo.components.animated.pressable :as animation]
+            [quo2.core :as quo2]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.multiaccounts.core :as multiaccounts]
+            [status-im.ui.screens.activity-center.notification.mentions.style :as style]
+            [status-im.ui.screens.activity-center.utils :as activity-center.utils]
+            [status-im.utils.datetime :as datetime]
+            [utils.re-frame :as rf]
+            [clojure.string :as str]))
+
+(defn message-body [message]
+  (let [parsed-text (get-in message [:content :parsed-text])
+        parsed-text-children (:children (first parsed-text))]
+    [text/text {:number-of-lines     2
+                :style               style/tag-text
+                :accessibility-label :activity-message-body
+                :size                :paragraph-1}
+     (for [{:keys [type literal]} parsed-text-children]
```

Comment:
It's a map (of text/message data) that has been iterated inside a Text Component. We return a new text component if it's a `mention` where we display the contact name (instead of the public key) with some additional text styles without affecting the parent text style and maintaining the same line as the parent. The rest of the text will be fed as it is (without any component/style).

Reply:
Okay, so are you sure there's no warning being generated by the for loop without a unique key? I thought `^{:key ...}` was mandatory here.

Reply:
I just tested locally and I'm getting a warning due to the collection being returned as children of quo2/text.

```
  ERROR    Warning: Each child in a list should have a unique "key" prop.

Check the render method of `quo2.components.markdown.text.text`. See https://fb.me/react-warning-keys for more information.
    in quo2.components.markdown.text.text (created by quo2.components.markdown.text.text)
    in quo2.components.markdown.text.text (created by quo2.components.notifications.activity_log.view.activity_message)
    in RCTView (at View.js:34)
    in View (created by quo2.components.notifications.activity_log.view.activity_message)
```

Reply:
My apologies - I will add the key 👍 :)